src/cclib/parser/molproparser.py

```diff
@@ -282,7 +313,9 @@ def extract(self, inputfile, line):
             line = next(inputfile)
             while line.strip():
                 temp = line.strip().split()
-                atomcoords.append([utils.convertor(float(x), "bohr", "Angstrom") for x in temp[3:6]])  # bohrs to angs
+                atomcoords.append(
+                    [utils.convertor(float(x), "bohr", "Angstrom")
+                     for x in temp[3:6]])
```

Comment:
I think it would have been sufficient to not break on the first append, and just break inside the list comprehension. Dangling open parens for method calls look funny.

Reply:
Again it exceeds then 80 characters